lib/src/page_list_viewport_gestures.dart

```diff
@@ -415,23 +623,95 @@ class DeprecatedPanAndScaleVelocityTracker {
       }
     }
 
-    if (_previousGesturePointerCount! > 1) {
+    if (_previousGesturePointerCount > 1) {
       // The user was scaling. Now the user is panning. We don't want scale
-      // gestures to contribute momentum, so we set the launch velocity to zero.
+      // gestures to contribute ballistic, so we set the launch velocity to zero.
       // If the panning continues long enough, then we'll use the panning
-      // velocity for momentum.
+      // velocity for ballistic.
       PageListViewportLogs.pagesListGestures
           .fine(() => " - this gesture was a scale gesture and user switched to panning. Resetting launch velocity.");
       _launchVelocity = Offset.zero;
       return;
     }
 
     final translationDistance = (_lastFocalPosition - _startFocalPosition).distance;
-    if (translationDistance > kViewportReAdjustmentMinTranslationDistance &&
-        velocity.distance < kViewportReAdjustmentMaxVelocity) {
-      // The user was readjusting the viewport by dragging it to the
-      // new position.
+    final velocityMagnitude = velocity.distance;
```

Comment:
I think I mentioned this before and I'm not sure if there was an answer. Why refer to something as "velocity magnitude" instead of calling it speed?

Reply:
There was an answer. Speed is a weird quantity, that physicists, for example, try not to use because it comes with a duality: is it average or instantaneous? So 2 words in either case: velocity magnitude or instantaneous speed.

Reply:
Actually velocity magnitude is 1 character shorter!